src/server.cpp

```diff
@@ -1069,19 +1069,19 @@ void Server::Receive(float timeout)
 		pkt.clear();
 		peer_id = 0;
 		try {
-			if (!m_con->ReceiveTimeoutMs(&pkt,
+			if (m_con->ReceiveTimeoutMs(&pkt,
```

Comment:
IMO a more intuitive change would be to use `continue;` if the time condition on L1082 does not apply. Early guards are commonly easier to read than nested code.

Reply:
depends on code style, but you're that Minetest largely follows the early exit style. I'll change it.